draft-ietf-jsonpath-base.md

```diff
@@ -97,6 +97,13 @@ informative:
     seriesinfo:
       ISO/IEC 22537:2006
     date: 2006
+  E4X-overview:
```

Comment:
I don't think we need this -- this seems to be a remnant from an older revision.

There are several such remnants, which make it hard to review this PR.